php/WC_Gateway_Blockonomics.php

```diff
@@ -182,6 +182,7 @@ public function check_blockonomics_callback()
 
         $callback_secret = get_option("blockonomics_callback_secret");
         $secret = isset($_REQUEST['secret']) ? $_REQUEST['secret'] : "";
+		$network_confirmations=get_option("blockonomics_network_confirmation");
```

Comment:
We can use a default value of 2. This is for upgrading users who have not clicked save after updating the plugin.

Reply:
Fixed with commit 145b3682e415450e612e3dff6efe184dbf0f9d48